beginners-yoga-course.html

```diff
@@ -22,14 +22,13 @@
 
 <!-- HEADER -->
 <div class="pageHeader__logo"></div>
-<div class="gridCell--row gridCell--padded gridCell--row--justifyCenter">
```

Comment:
Hahahaha haven't seen that in a while @spencerfdavis. Good to get the 👃 reinforcement.